City-of-Pori/pori-kada
# PORI-95 Attraction card styling

To test:

1. drush fra -y && drush cc all
2. Enable visitpori theme if it not enabled already.
3. Add an attraction card and make sure the layout looks nice like in the layout.

https://drive.google.com/drive/u/0/folders/0B7ZHsug7PcEDbEI5NVFjaVNBRm8

NOTICE: Does not include slideshow and icons for the accessibility info.
